vdom/helpers.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+vdom.helpers
+~~~~~~~~~
+
+Little helper functions for (many of) the HTML Elements.
+
+from vdom.helpers import div, img
+
+div(
+    img(src="http://bit.ly/storybot-vdom")
+)
+
+"""
+
+from .core import createComponent
+
+h1 = createComponent('h1')
+p = createComponent('p')
+div = createComponent('div')
+img = createComponent('img')
+b = createComponent('b')
+
+table = createComponent('table')
+thead = createComponent('thead')
```

Comment:
Perhaps @betatim should review this line :smile:

Reply:
Haha, didn't even think of that. I'm now going to refer to him at `<thead />`